projects/plugins/jetpack/class.jetpack.php

```diff
@@ -941,6 +941,31 @@ public function late_initialization() {
 		add_filter( 'map_meta_cap', array( $this, 'jetpack_custom_caps' ), 1, 2 );
 	}
 
+	/**
+	 * Checks if Jetpack Backup is active or waiting credentials.
+	 * Will return true if the state of Backup is anything except "unavailable".
+	 *
+	 * @return bool|int|mixed
+	 */
+	public static function should_show_backup() {
+		// Backup is a paid feature, therefore requires a user-level connection.
+		if ( ! static::connection()->has_connected_owner() ) {
+			return false;
+		}
+		$backup_enabled = get_transient( 'jetpack_rewind_state' );
```

Comment:
I'm checking for a different transient this time. This one is created by the admin menu.